### DISCUSSION TITLE
[@types/async-retry] No option for retries in @types/async-retry

### DISCUSSION THREAD

**Discussion by ghost** [DefinitelyTyped/DefinitelyTyped#61848] [Issues with a @types package]
No option for retries in @types/async-retry.
https://github.com/vercel/async-retry

My environment

```package.json
{
  "dependencies": {
    "async-retry": "^1.3.3"
  }
  "devDependencies": {
    "@types/async-retry": "^1.4.5"
  }
}
```

[a comment by an automated account is not shown]

**Comment by BendingBender** [comment]
@yogarasu Did you try out the setup from the [gist in this comment](https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/60193#discussioncomment-3433206)?
